### PR TITLE
Docs(web-twig): Introduce Helpers story refs, add Tabs to docs #DS-247

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

File: apps/web-twig-demo/src/Controller/HelpersController.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class HelpersController extends AbstractController
+{
+    #[Route('/')]
```
Comment (literat) [inline apps/web-twig-demo/src/Controller/HelpersController.php:11]:
```suggestion
    #[Route('/helpers')]
```

File: apps/web-twig-demo/templates/default.html.twig [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 {% block content %}
 
+  {% include 'partials/tabs.html.twig' %}
```
Comment (literat) [inline apps/web-twig-demo/templates/default.html.twig:7]: I think we should create a new layout for index pages where the header and the tabs are included. Thus we will be not repeating them on every page. What do you think?

File: apps/web-twig-demo/templates/partials/tabs.html.twig [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<ul class="Tabs mb-1100" role="tablist">
```
Comment (literat) [inline apps/web-twig-demo/templates/partials/tabs.html.twig:1]: Make the call only once `{% set routeName = app.current_route %}`.

https://symfony.com/doc/current/routing.html#getting-the-route-name-and-parameters

**Review by literat — CHANGES_REQUESTED** [review]
There are a few changes that I suggested. Feel free to create the new layout. That is not a blocking issue.

File: apps/web-twig-demo/templates/default.html.twig (hunk 2 above)
Comment (crishpeen) [reply apps/web-twig-demo/templates/default.html.twig]: It should be part of `default` and components&helpers should use `plain`. Fixed.

File: apps/web-twig-demo/templates/partials/tabs.html.twig (hunk 3 above)
Comment (crishpeen) [reply apps/web-twig-demo/templates/partials/tabs.html.twig:3]: > The app.current_route and app.current_route_parameters variables were introduced in Symfony 6.2. Before you had to access _route and _route_params request attributes using app.request.attributes.get().

😁

**Review by literat — APPROVED** [review]

File: apps/web-twig-demo/src/Controller/HelpersController.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    #[Route('/helpers')]
+    public function index(): Response
+    {
+        return $this->render('helpers.html.twig', ['helpers' => $this->getWebTwigHelpers()]);
+    }
+
+    /**
+     * @return Array<string>
+     */
+    private function getWebTwigHelpers(): array
```
Comment (adamkudrna) [inline apps/web-twig-demo/src/Controller/HelpersController.php:20]: Could be shared with `DefaultController` through an abstract controller maybe…? @literat

Comment (literat) [reply apps/web-twig-demo/src/Controller/HelpersController.php:20]: Yes, I think it can be shared via AbstractController.

**Pull request merged**